src/PantherTestCaseTrait.php

```diff
@@ -92,6 +92,11 @@ public static function tearDownAfterClass(): void
         }
     }
 
+    public static function getPrimaryPantherClient(): ?PantherClient
```

Comment:
Why this change? I'm not a fond of adding this method.

Reply:
I don't like it either but was the only way I could think of to get the current client in the PHPUnit extension. In [zenstruck/browser](https://github.com/zenstruck/browser), I sort of do the opposite: when the browser/clients are created, I [register with the extension](https://github.com/zenstruck/browser/blob/25b111e121abb38a91eb587434828bb987dc512e/src/Browser/Test/HasBrowser.php#L57). Would you like something like this or do you have another suggestion?

Reply:
I definitely prefer how you do in Browser! Can we switch to something like this?

Reply:
Sounds good. Another thing I do in browser is take screenshots for all browser instances used in the tests. I should do the same thing here, yes?

Reply:
Good idea!

Reply:
See 113fafe2fa5fb31c4f2bfe39c0e052ce695dedca, wdyt?

Reply:
LGTM!